src/qml/qgismobileapp.qml

```diff
@@ -2440,6 +2444,14 @@ ApplicationWindow {
         property bool jumpedOnce: false
 
         function jumpToLocation() {
+          // Callback to activate follow mode after jump completes
+          const activateFollowMode = function () {
+            if (!gnssButton.followActive) {
+              mapCanvasMap.freeze('follow');
```

Comment:
I suspect we'll need to force refresh here to insure that the soft/hard lock has a nice rendered map to work with (vs. an out of scale / bounds situation).